internal/config/benchmark.go

```diff
@@ -234,8 +234,8 @@ func (b *BenchmarkScenario) Bind() *BenchmarkScenario {
 
 // BenchmarkJobImageInfo represents the docker image information for benchmark job.
 type BenchmarkJobImageInfo struct {
-	Image      string `json:"image,omitempty"       yaml:"image"`
-	PullPolicy string `json:"pull_policy,omitempty" yaml:"pull_policy"`
+	Image      string `info:"image"       json:"image,omitempty"       yaml:"image"`
+	PullPolicy string `info:"pull_policy" json:"pull_policy,omitempty" yaml:"pull_policy"`
```

Comment:
Could you please explain how the `info` tag is used? 🙏

Reply:
I just added tags all at once and there is no use for it at the moment.
Should I delete it?

Reply:
Thank you for the explanation!
Yes, could you please delete it? 🙏